libtsuba/include/tsuba/ParquetReader.h

```diff
@@ -17,22 +23,67 @@ class KATANA_EXPORT ParquetReader {
     int64_t length;
   };
 
-  /// \returns a Reader that will read a table from storage location optionally
-  /// reading only part of the table defined by \param slice
+  struct ReadOpts {
+    /// if true (default) make sure canonical types are used and table columns
+    /// are not chunked
+    bool make_safe{true};
```

Comment:
perhaps we should call this `make_canonical` since we mean this in the sense that katana parquet files are a subset of all parquet files.